Makefile

```
@@ -16,6 +16,8 @@ HELM_OPTS ?= --set serviceAccount.annotations.eks\\.amazonaws\\.com/role-arn=${K
 			--set clusterEndpoint=${CLUSTER_ENDPOINT} \
 			--set aws.defaultInstanceProfile=KarpenterNodeInstanceProfile-${CLUSTER_NAME}
 
+CGO_ENABLED=0 # Disallows inclusion of CGO in the code and all dependencies
```

I believe you'll need to export to set this flag:

```suggestion
export CGO_ENABLED=0 # Disallows inclusion of CGO in the code and all dependencies
```

Can you move this up to the other compile related flag section of the Makefile as well?